### PR TITLE
Add a warning message for optimizer

### PR DESCRIPTION
**Pull request by daquexian**
ONNX Optimizer is moving to https://github.com/onnx/optimizer (https://github.com/onnx/optimizer/issues/1). A warning message should be added to suggest users use the new repo.

### CODE REVIEW

File: onnx/optimizer/optimize.cc [hunk 1]
```diff
@@ -11,6 +11,10 @@ GlobalPassRegistry Optimizer::passes;
 Optimizer::Optimizer(
     const std::vector<std::string>& names,
     const bool fixed_point) {
+  std::cout
+      << "WARNING: ONNX Optimizer has been moved to https://github.com/onnx/optimizer. "
```
Comment (askhade) [inline onnx/optimizer/optimize.cc:15]: Also add a statement stating that all further enhancements and fixes to optimizers will be done in this new repo.

**Comment by askhade** [comment]
Please add a similar message in the Optimizer documentation as well.

**Comment by daquexian** [comment]
@askhade Thanks! I have updated it.

**Comment by askhade** [comment]
@daquexian : Please fix the DCO failure.

**Review by askhade — APPROVED** [review]

**Comment by daquexian** [comment]
> @daquexian : Please fix the DCO failure.

Done

**Pull request merged**